iac-extensions/kubernetes/src/test/java/org/sonar/iac/kubernetes/plugin/KubernetesSensorTest.java

```diff
@@ -51,7 +51,10 @@ void shouldParseYamlFileWithIdentifiers() {
   @Test
   void shouldNotParseYamlFileWithHelmChartTemplate() {
     analyse(sensor(), inputFile(K8_IDENTIFIERS + "foo: {{ .bar }}"));
-    assertNotSourceFileIsParsed();
+    assertOneSourceFileIsParsed();
+
+    var logs = logTester.logs(Level.DEBUG);
+    assertThat(logs.get(0)).isEqualTo("Helm content detected in File 'k8.yaml'");
```

Comment:
```suggestion
    assertThat(logs).contains("Helm content detected in File 'k8.yaml'");
```
similar in next test case